app/models/lesson.rb

```diff
@@ -1,4 +1,8 @@
 # frozen_string_literal: true
 
 class Lesson < ApplicationRecord
```

Comment:
Mối quan hệ giữa lesson và word là quan hệ nhiều - nhiều nha em. 
Em đọc lại rails association xem cách người ta làm rồi áp dụng tương tự nè 
https://guides.rubyonrails.org/association_basics.html#the-has-many-through-association